src/main/resources/reference_points.xml

```diff
@@ -30,10 +30,22 @@
     <Image name="road_2lane_6right.png">
         <Refpoint x="350" y="367"/>
     </Image>
+    <Image name="car_1_white.png">
```

Comment:
Felmerül a kérdés, hogy a @SzFMV2020-Tavasz/team-a3 is az autó tömegközéppontját tekinti-e forgatási referenciapontnak.

Reply:
A kérdés jó, de 
- egocar szempontjából úgy van megírva a kód (VisualizationConfig), hogy nagyon könnyen változtatjuk a forgatási középpontot.  (az egy külön változó, ez itt csak az általános képfelhasználás esetén van
- a car_2... variáciüknál mindegyiknél így volt, van esélye, hogy a car_1 variációknál nem így lesz?

Reply:
Az, hogy hogy lesz rajtatok múlik, így is lehet. Itt tényleg a konszenzus meglétére hívtam csak fel a figyelmet.